lib/std/start.zig

```diff
@@ -105,7 +105,7 @@ fn main2() callconv(.C) c_int {
     return 0;
 }
 
-fn _start2() noreturn {
+fn _start2() callconv(.C) noreturn {
```

Comment:
I'm still unsure about this, `callconv(.Naked)` doesn't work because the function contains a function call.

Reply:
Since this clearly isn't a long-term solution, anything that works well enough to unblock progress is perfectly fine.